compiler/optimizer/GeneralLoopUnroller.cpp

```diff
@@ -537,6 +533,8 @@ void TR_LoopUnroller::modifyBranchTree(TR_RegionStructure *loop,
 
    if (originalLimitNode->getOpCode().isLoadConst() && !const_overflow)
       {
+      // Address compare generated by loop strider does not have constant node,
+      // thus does not fall into this category
```

Comment:
I'm not sure this comment and the one above add much value. If there is an assumption about the nodes an assert is better or at the very least what case doesn't fall in here should be spelled out and not defined by the implementation of another opt which could change independently.